tests/framework-tests/core/js/ModelTransformationTests.js

```diff
@@ -59,29 +59,68 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         };
     };
 
-    fluid.tests.transforms.testOneTransform = function (message, model, transform, method, expected, transformWrap) {
-        if (transformWrap) {
-            transform = fluid.tests.transforms.wrapTransform(transform);
-        }
-        var transformed = fluid.model.transform(model, transform);
-        jqUnit[method].apply(null, [message, expected, (transformWrap ? transformed.value : transformed) ]);
+    fluid.tests.transforms.testOneTransform = function (test) {
+        var transformed = fluid.model.transform(test.model, test.transform);
```

Comment:
This change is responsible for the test failures reported at http://lists.idrc.ocad.ca/pipermail/fluid-work/2016-May/009969.html